enhancements/baremetal/bootstrap-external-ip.md

```diff
@@ -0,0 +1,126 @@
+---
+title: bootstrap-external-ip
+authors:
+  - "@honza"
+reviewers:
+  - "@hardys"
+  - "@dtantsur"
+  - "@dhellmann"
+  - "@patrickdillon"
+  - TBD
+approvers:
+  - TBD
+api-approvers:
+  - TBD
+creation-date: 2022-03-23
+last-updated: 2022-03-23
+tracking-link:
+  - https://issues.redhat.com/browse/METAL-175
+see-also:
+  - https://bugzilla.redhat.com/show_bug.cgi?id=2048600#c1
+replaces:
+superseded-by:
+---
+
+# Bootstrap External IP
+
+## Summary
+
+When installing a bare metal IPI cluster, you can use the `networkConfig` field
+in the `install-config.yaml` file to configure the control plane network
+interfaces for the cluster hosts.  However, currently, you cannot configure the
+bootstrap VM networking using the same means.
+
+## Motivation
+
+In environments where no DHCP server is running, the bootstrap VM will not get
+an IP address on the control plane NIC causing cluster installation to fail.
+The user can work around this by modifying the bootstrap ignition file; however,
+this isn't a friendly experience.
+
+### Goals
+
+- Support cluster installation in environments without a running DHCP server
+- Improve the level of customization of bootstrap VM networking
+- Improve user-friendliness when specifying a static IP for the bootstrap VM
+
+### Non-Goals
+
+- Provide means for configuring every aspect of bootstrap networking
+
+## Proposal
+
+We will add two new fields, `bootstrapExternalStaticIP` and
+`bootstrapExternalGateway`, to allow for further customization of bootstrap
+networking.  We will use these fields to generate static configuration in the
+installer.
+
+### User Stories
+
+- As a user, I want to be able to deploy an IPI bare metal cluster in an
+  environment where no DHCP cluster is running
+- As a user, I want to specify a static IP address for the bootstrap VM
+
+### API Extensions
+
+We will add two new fields to the `install-config.yaml` file called
+`bootstrapExternalStaticIP`, and `bootstrapExternalGateway`.  This is similar to
+the existing `bootstrapProvisioningIP` field.
+
+### Risks and Mitigations
+
+## Design Details
+
+### Test Plan
+
```

Comment:
We should follow up and add CI coverage for static IP deployments, I know @derekhiggins was [working on that](https://github.com/openshift-metal3/dev-scripts/pull/1342) but I'm not sure of the latest status